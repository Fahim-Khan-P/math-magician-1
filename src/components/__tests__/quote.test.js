import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from '../quote';

it('Quote page is rendering correctly', () => {
  const tree = renderer
    .create(<Quotes />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
