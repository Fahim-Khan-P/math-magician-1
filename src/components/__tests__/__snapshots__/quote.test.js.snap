// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote page is rendering correctly 1`] = `
<p
  className="quote"
>
  I can count all the lovers I have had on one hand...if I am holding a calculator.
</p>
`;
